src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -962,6 +963,15 @@ class XrdAccSciTokens : public XrdAccAuthorize, public XrdSciTokensHelper,
         }
         m_log.Emsg("Config", "Logging levels enabled -", LogMaskToString(m_log.getMsgMask()).c_str());
 
+        auto xrdEnv = static_cast<XrdOucEnv*>(envP ? envP->GetPtr("xrdEnv*") : nullptr);
+        auto tlsCtx = static_cast<XrdTlsContext*>(xrdEnv ? xrdEnv->GetPtr("XrdTlsContext*") : nullptr);
+        if (tlsCtx) {
+            auto params = tlsCtx->GetParams();
+            if (params && !params->cafile.empty()) {
+                scitoken_config_set_str("tls.ca_file", params->cafile.c_str(), nullptr);
```

Comment:
@bbockelm Given that we support these older Debian/Ubuntu distributions, and scitokens cannot be updated there, I think this part of the patch will need to be `#ifdef`ed out depending on the version of scitokens found. Can you please make a patch for that? Thanks!